core/kernel/console.c

```diff
@@ -57,16 +69,21 @@ void configure_console_from_dt(void)
 	char *stdout_data;
 	const char *uart;
 	const char *parms = NULL;
-	void *fdt = get_dt_blob();
-	int offs;
+	void *fdt;
+	int offs = -1;
 	char *p;
 
-	if (!fdt)
-		return;
-
-	offs = fdt_path_offset(fdt, "/secure-chosen");
+	/* Probe console from secure DT and fallback to non secure DT */
+	fdt = get_sec_dt_blob();
+	if (fdt)
```

Comment:
Test not needed

Reply:
I will preserve this. If there is no secure DTB we don't want to abort.

Reply:
Abort? `find_chosen_node(fdt)` returns -1 if `fdt` is NULL.